os/binfmt/binfmt_execmodule.c

```diff
@@ -362,6 +363,10 @@ int exec_module(FAR struct binary_s *binp)
 	} else {
 		BIN_RTTYPE(binary_idx) = BINARY_TYPE_NONREALTIME;
 	}
+	strncpy(BIN_VER(binary_idx), binp->bin_ver, BIN_VER_MAX);
```

Comment:
it seems that include important info, so validation checking is necessary, In my opinion.